.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Migrate code style to Black
+327835fae527bc63159a8bdd904b1546594e0329
```

Comment:
Can you do this in the same commit as the reformatting? Isn't that predicting the future then? (Kudos if you can pull that off)

Reply:
Overeager rebase and squashing 🤦. I'll move this to its own commit to avoid having to compute the hash.